pkg/clustersql/settings_test.go

```diff
@@ -0,0 +1,245 @@
+/*
+Copyright 2021 The Cockroach Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    https://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package clustersql_test
```

Comment:
I think are doing the same package name as the actu code.  I get why you did this, but for conformity should we just use the same names.  Should we remove `_test`?

Reply:
The `_test` suffix is currently being used in some of our packages already (see `controller`, `labels`, and `resource` for example). If we were to standardize, do you have a preference for using the same package?

I tend to prefer the `_test` suffix so that we're testing what an external caller would see.